File: src/com/rogerio/KillFly/Hero.java
package com.rogerio.KillFly;
/*
 * Autor: Rogerio C. Santos - rogerio@rogeriocs.com.br
 * http://www.rogeriocs.com.br
 */
import libs.com.rogerio.utils.Sprite;
import android.content.Context;
import android.graphics.Bitmap;
import android.graphics.BitmapFactory;

public class Hero extends SpritesObjs {
	public Hero(Context context) throws Exception{
		this.mContext=context;
		mSprite = new Sprite();
		Bitmap bmp = BitmapFactory.decodeResource(context.getResources(),R.drawable.bowl);
		mSprite.init(bmp, bmp.getHeight(), bmp.getWidth(), 1, 1);
	}
}

File: app/src/main/java/com/rogerio/KillFly/Hero.kt
package com.rogerio.KillFly

/*
 * Autor: Rogerio C. Santos - rogerio@rogeriocs.com.br
 * http://www.rogeriocs.com.br
 */
import android.content.Context
import android.graphics.BitmapFactory
import libs.com.rogerio.utils.Sprite

class Hero
constructor(context: Context) : SpritesObjs() {
    init {
        this.mContext = context
        sprite = Sprite()
        val bmp = BitmapFactory.decodeResource(context.resources, R.drawable.bowl)
        sprite!!.init(bmp, bmp.height, bmp.width, 1, 1)
    }
}
